README.md

```diff
@@ -15,3 +15,76 @@ the Elastic Search from acceptance to localhost with :
     ssh -L 9200:es01.acc.data.amsterdam.nl:9200 user@dc01.acc.data.amsterdam.nl
 
 Currently the Elastic Search index for Drupal is not in a backup and cannot yet be restored with update-el.sh.
+
+## How to add a map layer
+
+The json files for collections and layers can be found [here](./assets/map-collections.config.json) and [here](./assets/map-layers.config.json).
+Make sure you know to which collection you should add the layers. If the collection already exist,
+simply add an object with an `id` to the `mapLayers` array. This `id` can be made up, and should correspond to the `id` of the layer
+you're about to add. If the collection doesn't exist, read the instructions below.
+
+Now that we added an `id` to the collection, we have to add the layer. Open [map-layers.config.json](./assets/map-layers.config.json) and add the following to the array:
+
+```json
+{
+  "id": "someid",
+  "layers": ["some_layer"],
+  "title": "My Layer",
+  "type": "wms",
+  "url": "/maps/some_layer",
+  "legendItems": [
+    {
+      "title": "Not available",
+      "imageRule": "Niet beschikbaar"
+    },
+    {
+      "title": "Available"
+    }
+  ],
+  "meta": {
+    "description": null,
+    "themes": ["foo", "bar"],
+    "datasetIds": [],
+    "thumbnail": null,
+    "date": null
+  }
+}
+```
+
+- _id_ - this is the ID you just added to one of the collections in [map-collections.config.json](./assets/map-collections.config.json)
+- _layers_ - an array of the layers available in the `.map` file. These `.map` files can be found in [this repo](https://github.com/Amsterdam/mapserver/)
+- _title_ - this is the title that will be displayed in the front-end.
+- _type_ - usually just `wms`
+- _url_ - this is the url to retrieve the actual (wms) layer to be displayed on the map, but it's also used to retrieve the legend images (which are specified in the `legendItems` field)
+- _legendItems_ - an array of objects containing a `title`, that will be displayed in the frontend but can also be used to retrieve the image for the legens from the mapserver (checkout the corresponding `.map` file in [this repo](https://github.com/Amsterdam/mapserver/))
```

Comment:
Mention the `imageRule` here? 

The field `title` is used by the mapserver to create an icon for this layer or legend item. If the title doesn't match the title used in the `.map` file (for redactional purposes), the `imageRule` field can be used to define the legend icon.

Reply:
Will change description 👍  imageRule is mentioned below btw

Reply:
Ah overlooked that one :)